From c67238d4dae294723836bb18f15eb0ca63c50c6e Mon Sep 17 00:00:00 2001
From: Sandor Yu <Sandor.yu@nxp.com>
Date: Wed, 14 Sep 2016 19:47:50 +0800
Subject: [PATCH 1158/5242] MLK-13237: dts: Add vadc to i.MX6SX display power
 domain

commit  6e99718257af443c19530389d764add133831192 from
https://source.codeaurora.org/external/imx/linux-imx.git

VADC module is in i.MX6SX display power domain.
It should enabled with display power domain,
otherwise vadc will fail to be work when
display in low power mode.

Signed-off-by: Sandor Yu <Sandor.yu@nxp.com>
(cherry picked from commit 8ab2cc06a6900889bd61e8de6f5436523e8227d2)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sx.dtsi |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx6sx.dtsi b/arch/arm/boot/dts/imx6sx.dtsi
index 3dd07c6..d53b4f0 100644
--- a/arch/arm/boot/dts/imx6sx.dtsi
+++ b/arch/arm/boot/dts/imx6sx.dtsi
@@ -1424,6 +1424,7 @@
 					clocks = <&clks IMX6SX_CLK_VADC>,
 						 <&clks IMX6SX_CLK_CSI>;
 					clock-names = "vadc", "csi";
+					power-domains = <&gpc 2>;
 					gpr = <&gpr>;
 					status = "disabled";
 				};
-- 
1.7.9.5

